department-of-veterans-affairs/vets-website
# Add save button to spouse former marriage

## Summary

For all single-page list and loops on the form, there is a "save" button for the most recent entry. The [Spouse's former marriages](https://staging.va.gov/pension/apply-for-veteran-pension-form-21p-527ez/household/marital-status/spouse-marriages) page did not have this button. This PR adds it.

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/82848

## Testing done

- Visit https://staging.va.gov/pension/apply-for-veteran-pension-form-21p-527ez/household/marital-status/spouse-marriages
     - Enter 'married' on https://staging.va.gov/pension/apply-for-veteran-pension-form-21p-527ez/household/marital-status and 'yes' on https://staging.va.gov/pension/apply-for-veteran-pension-form-21p-527ez/household/marital-status/spouse-marital-history to access
     - You should see a save button
- Fill all required fields for the first spouse, then click 'save'. It should collapse.
- Click 'Add another former marriage of spouse'. The new spouse should have a save button and a remove button.

## Screenshots

|         | Before | After |
| ------- | ------ | ----- |
| Desktop |    <img width="546" alt="Screenshot 2024-05-14 at 1 42 44 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/bfe9deaa-6856-4faa-b353-117b61724359">    |    <img width="550" alt="Screenshot 2024-05-14 at 1 48 22 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/21046714/d1732ad5-7e4b-4544-8b9a-05f98bbb6114">   |

## What areas of the site does it impact?

Pension Benefits

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication
